README.md

```diff
@@ -4,57 +4,53 @@ Maestro is a tool for managing and running AI agents and workflows.
 
 ## Installation
 
-1. Clone the repository:
 ```bash
-git clone https://github.com/AI4quantum/maestro.git
-cd maestro
-```
-
-2. Install dependencies:
-```bash
-uv venv
-source .venv/bin/activate
-uv pip install -r requirements.txt
-uv pip install -e .
+pip install git+https://github.com/AI4quantum/maestro.git@v0.1.0
 ```
 
-Note: If using scoring agents, additionally install:
+Note: If using crewai agents, install:
```

Comment:
Actually it's the scoring agent... check with @george-lhj

Reply:
I had originally called the extra `scoring` as seen in my demo Monday, but when reading the code further the warning was coming from the `CrewAIAgent` so I changed it.

@george-lhj which is correct? Either way I think naming the extra package `crewai` is better since that's what is actually installs